### ISSUE TITLE
AI: meaning of PresenceType in autonomyType

### ISSUE THREAD

**Issue by bact** [spdx/spdx-3-model#719]
[`autonomyType`](https://github.com/spdx/spdx-3-model/blob/main/model/AI/Properties/autonomyType.md) in `AIPackage` has a type of [`/Core/PresenceType`](https://github.com/spdx/spdx-3-model/blob/main/model/Core/Vocabularies/PresenceType.md).

`/Core/PresenceType` has 3 possible values:
- yes: Indicates presence of the field.
- no: Indicates absence of the field.
- noAssertion: Makes no assertion about the field. 
 
This is a summary of `autonomyType`:

    States if a human is involved in the decisions of the AI software.

If a human is involved, should `autonomyType` be "yes" or "no"?

In other words, does "Yes" means:
1. "Yes, there is human involved" (refer to the summary: "if a human is involved ...") or
2. "Yes, the system is autonomous" (refer to the name of the field: "autonomyType") ?
 
May need a further clarification in the property description. ("Yes is..., No is ...")

Similar question for:
- [/AI/Properties/useSensitivePersonalInformation](https://github.com/spdx/spdx-3-model/blob/main/model/AI/Properties/useSensitivePersonalInformation.md)
- [/Dataset/Properties/hasSensitivePersonalInformation](https://github.com/spdx/spdx-3-model/blob/main/model/Dataset/Properties/hasSensitivePersonalInformation.md)

**Comment by bennetkl** [comment]
I saw it as "yes" when algorithm can perform the programmed operations  without human input or guidance.

**Comment by bact** [comment]
@bennetkl I agree with that interpretation - the semantic is straightforward to the name of the field.

But with that, we may need to rewrite the summary to something like this instead:

"States if the system can make decision or take action without the involvement of a human."

**Comment by bact** [comment]
Way forward to 3.1, we may like to consider that automation has degrees.

See this table of varying degrees of automation from https://doi.org/10.3390/ijerph19063641 :

![Screenshot 2024-04-24 at 13 38 35](https://github.com/spdx/spdx-3-model/assets/128572/fd1acc34-c930-457d-ad42-2532688b7bf8)

**Comment by bact** [comment]
As a note, I feel that the use of `PresenceType` here for "human presence" is a little bit of mismatch.

To my understanding, the type is more about presence of a metadata.

Its description says:

```text
This type is used to indicate if a given field is present or absent or unknown.
```

**Comment by bact** [comment]
This issue may resolved by #741
